.travis.yml

```diff
@@ -1,14 +1,19 @@
 language: python
-python:
-  - "3.5"
-  - "3.6"
-  - "3.7"
-  - "3.8"
+matrix:
+  include:
+    - python: 3.5
+      env: TOXENV=py35
+    - python: 3.6
+      env: TOXENV=py36
+    - python: 3.7
+      env: TOXENV=py37
+    - python: 3.8
+      env: TOXENV=py38
+    - env: TOXENV=flake8
 install:
-  - pip install codecov flake8
+  - pip install tox
 script:
-  - flake8
-  - coverage run --source github,tests setup.py test
```

Comment:
Does this mean we lose coverage reports?

Reply:
Thanks for the feedback. Again this PR is specifically in Draft mode, which means it's not yet ready for review

Reply:
It's never early for reviews! :-)

Reply:
The draft mode was mean to not disturb the reviewers 😬